Examples/Examples.xcodeproj/project.pbxproj

```diff
@@ -411,7 +411,7 @@
 				SUPPORTS_MACCATALYST = YES;
 				SUPPORTS_MAC_DESIGNED_FOR_IPHONE_IPAD = NO;
 				SWIFT_VERSION = 5.0;
-				TARGETED_DEVICE_FAMILY = "1,2";
+				TARGETED_DEVICE_FAMILY = "1,2,6";
```

Comment:
I think adding Mac as a target device makes sense, though I had a hard time finding any Apple doc on how to set this for an iOS app that supports Mac Catalyst. 🤔